### PR TITLE
Fix govet

### PR DESCRIPTION
**Pull request by mfojtik**
See: https://github.com/openshift/origin/pull/8398

### CODE REVIEW

**Comment by soltysh** [comment]
LGTM

**Comment by mfojtik** [comment]
[merge]

File: hack/verify-govet.sh [hunk 1]
```diff
@@ -21,8 +21,7 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false \
-              $test_dir
+  go vet -shadow=false $test_dir
```
Comment (kargakis) [inline hack/verify-govet.sh]: It doesn't seem that `go vet` can use its flags - you still need to use `go tool vet`. Also -shadow is by default false.

Comment (kargakis) [inline hack/verify-govet.sh]: Meaning you can drop the flag and still use `go vet`?

Comment (mfojtik) [inline hack/verify-govet.sh]: sure :-)

File: hack/install-tools.sh [hunk 2]
```diff
@@ -8,4 +8,4 @@ source "${STI_ROOT}/hack/common.sh"
 GO_VERSION=($(go version))
 echo "Detected go version: $(go version)"
 
-go get golang.org/x/tools/cmd/cover github.com/golang/lint/golint golang.org/x/tools/cmd/vet
+go get golang.org/x/tools/cmd/cover github.com/tools/godep
```
Comment (rhcarvalho) [inline hack/install-tools.sh]: Why was golint removed?

**Pull request merged**